src/main/resources/assets/immersiveengineering/lang/zh_CN.lang

```diff
@@ -277,14 +280,15 @@ chat.ImmersiveEngineering.warning.tooFar=你离上一个连接点太远了
 chat.ImmersiveEngineering.warning.cantSee=连接被阻断
 chat.ImmersiveEngineering.warning.connectionExists=连接已存在
 chat.ImmersiveEngineering.warning.invalidPoint=存在无效连接点
-chat.ImmersiveEngineering.info.averageLoss=两点间平均能量损耗: %1$s RF
+chat.ImmersiveEngineering.info.averageLoss=两点间平均能量损耗: %1$s %
 chat.ImmersiveEngineering.info.energyStorage=存储能量: %1$s/%2$s RF
 
-chat.ImmersiveEngineering.info.forChunk=§8For Chunk from %1$s to %2$s
+chat.ImmersiveEngineering.info.forChunk=§8对于位于 %1$s 至 %2$s 的区块
```

Comment:
@BluSunrize If my understand is right, here, *Chunk* shall not be translated to ["区块"](http://minecraft.gamepedia.com/Chunks), because the "Chunk" here refers to a big area with certain range from somewhere (`%1$s`) to somewhere else (`%2$s`), and that is different from a *chunk* in Minecraft with fixed 16 by 16 by 256 blocks.
If I am wrong please ignore it.